pvlib/irradiance.py

```diff
@@ -661,9 +670,23 @@ def isotropic(surface_tilt, dhi):
         <=180. The tilt angle is defined as degrees from horizontal
         (e.g. surface facing up = 0, surface facing horizon = 90)
 
+    surface_azimuth : numeric
+        Surface azimuth angles in decimal degrees. surface_azimuth must
+        be >=0 and <=360. The azimuth convention is defined as degrees
+        east of north (e.g. North = 0, South=180 East = 90, West = 270).
+
     dhi : numeric
         Diffuse horizontal irradiance in W/m^2. DHI must be >=0.
 
+    horizon_azimuths: numeric (optional)
+        Azimuth values for points that define the horizon profile. The ith
+        element in this array corresponds to the ith element in horizon_angles.
+
+    horizon_angle: numeric (optional)
+        Dip angle values for points that define the horizon profile. The ith
```

Comment:
Would help to define 'dip angle' here.

Reply:
Perhaps this might be a good place to comment on the term `dip angle`.  If I understand correctly, a typical horizon will consist of negative dip angles.  This seems counter-intuitive to me and contrary to usage I've seen in other places, where the horizon has a height (also an angle of course).  I am curious how the term `dip` came to be used.

Reply:
me too

Reply:
I do agree that dip angle is a confusing term. Even more confusing is that I am not really using "true" dip angles in the module since I treat a typical horizon as having positive dip angles (I use dip angle to mean degrees above the horizontal). The term came from http://sherrytowers.com/2014/04/13/archeoastronomy-calculating-the-horizon-profile-using-online-us-geographic-survey-data/ which I had been following when first starting work on horizon code. 
What would be a better term to use? Horizon elevation angle?

Reply:
I'm ok with "dip angle" but whatever term is used it needs a definition.

Reply:
changed dip angles to elevation angles. I think it makes more sense since its given in degrees above the horizontal